Tests/StorageApiHandler.php

```diff
@@ -70,7 +70,7 @@ public function getVerbosity()
     /**
      * @inheritdoc
      */
-    public function handle(array $record)
+    public function handle(array $record): bool
```

Comment:
Náhodou jsem řešil vedle https://github.com/keboola/docker-bundle/pull/637
Možná bych upravil i závislosti v composer.json?

Reply:
Jak upravil? Tohle pojede i se starsi verzi Monologu - jen to zpresnuje return type. Plati tam takova ta kontravariance nebo kovariance.

Reply:
Mas pravdu, to mi nedoslo. Tak me ignoruj :-)